config/v1/types_build.go

```diff
@@ -72,6 +76,28 @@ type ImageLabel struct {
 	Value string `json:"value,omitempty"`
 }
 
+type RegistriesConfig struct {
+	// SearchRegistries lists the registries to search for images if an image repository is not specified in an image pull spec.
+	//
+	// If this is not set, builds will search Docker Hub (docker.io) when a repository is not specified.
+	// Setting this to an empty list will require all builds to fully qualify their image pull specs.
+	// +optional
+	SearchRegistries *[]string `json:"searchRegistries,omitempty"`
```

Comment:
i thought in the past we could only configure this as a docker daemon setting.
i am wondering how many customers would set the search registry differently for build time versus runtime.
the runtime capability is not there pending https://jira.coreos.com/browse/RUN-207 , but why would an admin want to configure this twice?

Reply:
I spoke w @bparees and given that this is isolated to build time and not runtime and/or kubelet behavior, I am fine with this.

Reply:
echoing what was discussed in slack - this gives us flexibility to allow admins to say how developers define their pull specs. 

Ex: builds use the default docker.io search so their Dockerfiles don't break with a `FROM centos` declaration, but runtime disables search so every image running on the cluster declares where it is pulled from.